### PR TITLE
BACK-58 

### PR DESCRIPTION
**Pull request by Manizha-khorram**
Setup WebSocket server connection
@jpremo following this link => https://medium.com/@netralizmefod/how-to-test-server-socketio-with-postman-fd8c48852c7d
I tested this in postman and got (New client connected!) message!
Thanks to @larasodjati  helped me to figure it out.

### CODE REVIEW

**Comment by larasodjati** [comment]
You're very welcome, @Manizha-khorram. You did a great job setting up the socket io in our server! It's confusing for me to read the documentation but you made me understand how it works with your code :).

@jpremo I have tested it on the Postman, and I successfully can connect to our server and disconnect from it as well. Could you please take a look and approve it if everything's ok for you as well? Thank you.

File: src/server.js [hunk 1]
```diff
@@ -1,11 +1,18 @@
 const app = require('./app')
 const connectDB = require('../db/connect')
+const http = require('http')
+const intiatSocket = require('./socket')
```
Comment (jpremo) [inline src/server.js:4]: I think this should be spelled as initiateSocket. If you update this here, be sure to update all other references.

**Review by jpremo — APPROVED** [review]
Nice work on this! Especially figuring out how to test. I think there is just one spelling error. I'm going to approve this. Feel free to merge after fixing the spelling issue

File: src/server.js (hunk 1 above)
Comment (Manizha-khorram) [reply src/server.js]: Will do Jacob, thanks alot for the review.

**Pull request merged**